foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -403,12 +408,18 @@ class CoreStartCompletionCallback : Java.Lang.Object, IAdobeCallback
 
 ### Implement AEP Assurance session start APIs \(iOS only\)
 
-The `startSession` API needs to be called to begin a Project Griffon session. When called, SDK displays a PIN authentication overlay to begin a session.
+(session start APIs for other platforms listed [here](https://aep-sdks.gitbook.io/docs/foundation-extensions/adobe-experience-platform-assurance/assurance-api-reference) )
+
+The [startSession](./assurance-api-reference.md#startsession) API needs to be called to begin a Project Griffon session. When called, SDK displays a PIN authentication overlay to begin a session.
 
 {% hint style="info" %}
+<<<<<<< HEAD
+You may call this API when the app launches with a url (see code snippet below for sample usage)
+=======
 The AEP Assurance Android extension does not require this API to be called since it registers the app lifecycle handlers which automatically pick up any deep links and use them to start the session.
 
 When using the AEP Assurance iOS extension, you should call this API when the app launches with a url (see code snippet below for sample usage).
+>>>>>>> 07944957686b267a7744c803e262d07c960847fd
```

Comment:
Can you please remove L416, L418, L422 as they seem to be the result of the github merge